back/src/api/common/services/alcohol/alcoholTaxCalculator.ts

```
@@ -0,0 +1,146 @@
+  ): AlcoholTaxDetail[] {
+    const alcoholProducts = this.filterAlcoholProducts(detailedShoppingProducts);
+    const groupedByType = this.groupByType(alcoholProducts);
+
+    return Object.entries(groupedByType).map(([type, products]) => {
+      const mappedType = this.mapType(type);
+      const rates = this.TAX_RATES[mappedType as keyof typeof this.TAX_RATES];
+
+      if (!rates) {
+        console.warn(`Unknown alcohol type: ${type} (mapped to ${mappedType})`);
+        return {
+          type: this.getReadableTypeName(type),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)